lib/controls/checkbox/index.tsx

```diff
@@ -7,18 +7,43 @@ type OwnProps = React.HTMLProps<HTMLInputElement> & {
   isStandard?: boolean;
 };
 
-function CheckboxControl({ className, isStandard, ...props }: OwnProps) {
+function CheckboxControl({
+  className,
+  isStandard,
+  checked,
+  ...props
+}: OwnProps) {
   return (
     <span
       className={classNames('checkbox-control', [
         className,
         { 'checkbox-standard': isStandard },
       ])}
     >
-      <input type="checkbox" {...props} />
-      <span className="checkbox-control-base">
-        <span className="checkbox-control-checked" />
-      </span>
+      <input type="checkbox" {...props} checked={checked} />
+      {!isStandard ? (
+        <span className="checkbox-control-base">
+          <span className="checkbox-control-checked" />
+        </span>
+      ) : checked ? (
+        <svg
```

Comment:
Should we create components for them as we do the icons: https://github.com/Automattic/simplenote-electron/blob/develop/lib/icons/connection.tsx

The folder name doesn't make a ton of sense. But the contents are all svg's.

Reply:
That's a good call, it cleans up the checkbox component quite a bit by creating separate components for the svgs. I've done that now.